lib/ansible/modules/setup.py

```diff
@@ -133,7 +133,7 @@
       - '!<any valid subset>'
       - facter
 
-- name: Collect only selected facts
+- name: Display only selected facts
```

Comment:
`Display` isn't really the right term here, because the module does nothing to display anything.  The examples above these tasks are running adhoc commands, which will display the results, whereas a normal task isn't intended to accomplish that.  I'd argue still that `collect` is still appropriate.  The `setup` module will still query all of the subsets specified (all if none), but then restrict what it returns to those in `filter`.  Ultimately the controller only "collects" what the module returns.

`gather_subset` restricts what subsets are queried, `filter` does post query filtering of keys.

But maybe we should stick to using the names of the module arguments here, and just call it "Filter facts"

Reply:
Okay, I see. In that case, I agree, display is definitely not right here. I thought of using 'filter facts'; however it seems redundant. What about something like 'returns'? As in the description of the filter keyword: "If supplied, only _return_ facts that match one of the shell-style (fnmatch) pattern."

Collects implies that it _only_ collects those that match the filter (i.e. filtering happens during collection not post).

Reply:
```suggestion
- name: Filter and return only selected facts
```